site-src/index.md

```diff
@@ -20,54 +20,119 @@ variety of networking use-cases.
+permits (or denies) specific teams to bind to this Gateway and expose their
+applications through it. Centralized policies [such as
+TLS](tls.md#downstream-tls) can be enforced on the Gateway by the cluster
+operator. Meanwhile, the store and site teams run [in their own
+Namespaces](multiple-ns.md), but bind their Routes against the same shared
+Gateway, allowing them to independently control their [routing
+logic](http-routing.md). This separation of concerns allows the store team to
+manage their own [traffic splitting rollout](traffic-splitting.md) while
+leaving centralized policies and control to the cluster operators.
+
+![Gateway API Roles](./images/gateway-roles.png)
+
+While the Gateway API standardizes the boundaries of its resources, it
+provides flexibility within the API structure for policy & configuration
+attachment to be attached at different levels to adapt to the needs of  an
+organization. Deploy shared Gateways for more control or allow developers to
+[deploy their own dedicated  Gateways](api-overview.md#route-binding) for
+maximum autonomy. The API is designed with boundaries, which are also designed
+to be crossed, if that fits your model. 
+
+
+### Why does a portable API matter?
+
+Portability increases familiarity for new users, reduces the complexity of
+using multiple implementations, allows higher-level tooling & products to
+build on top of many implementations more easily, and it reduces the need for
+Gateway implementers to build custom (yet oddly similar) APIs. There are
+diverse benefits for both the users and implementers of a portable and
+standardized API. However, deep and meaningful portability is challenging.
+Portability is a constant choice between consistent behavior and the unique &
+valuable capabilities of a specific implementation - how to provide an API
+that does both? 
+
+The Gateway API is designed to strike a balance between these two sides
+through flexible conformance and extensibility. Gateway defines [multiple
+levels of conformance](guidelines.md#conformance), which provides implementers
+with flexibility in how much of the Gateway API they want to support. In
+combination with various extension points within the API, this allows the core
+functionality of Gateway to be portable, while still surfacing
+implementation-specific capabilities.
+
+### Who is working on Gateway?
+
+The Gateway API is a Kubernetes
+[SIG-Network](https://github.com/kubernetes/community/tree/master/sig-network)
+project being built by a variety of companies and users. Current and
+in-progress implementations include Contour, Google Kubernetes Engine (GKE),
+Istio, Kong, and Traefik. Check out the [implementations
+reference](implementations.md) to see the latest projects & products that
+support Gateway. If you are interested in contributing to or building an
+implementation using the Gateway API then don’t hesitate to [get
+involved!](#how-to-get-involved)
```

Comment:
This has to match section header. I don't have a strong preference between the two, but since the section header is changing here, this should to.
```suggestion
involved!](#getting-involved)
```